### PR TITLE
move the inputs on a different line then their label, adjust button t…

### LINKED ISSUE #64: Improvement: line up search button and alter labels in index page
As developer, to better organize file index alter labels. As a user, see the search button match with the rest of the document.

### PR DESCRIPTION
**Pull request by fab514**
…o match rest of layout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,11 +49,10 @@ <h1 class="textheader">Do You Want to Do Something?</h1>
                         <option value="diy">DIY</option>
                         <option value="music">Music</option>
                     </select>
-                </label>
-            </div>
-            <div class="button">
-                <button class="searchButton">Search</button>
-            </div>
+                </div>
+                <div class="button">
```
Comment (Sonic12040) [inline index.html]: I think you can get rid of this wrapper and just style the button itself, and I would say take advantage of the color scheme that you have come up with for this site and the visual language of the other elements on the page.
What does the button look like when it isn't being used? What about when you hover over it? What about while you're clicking on it?

File: index.html [hunk 2]
```diff
@@ -23,18 +23,20 @@ <h1 class="textheader">Do You Want to Do Something?</h1>
...
+            <div class="input" name="input">
+                <input type="number" name="participants" value="1" min="1" max="100">
             </div>
-            <div class="type">
-                <label>
+            <div class="type" name="type">
+                <label name="label">
                     Activity Type
+                </label>
+                <div name="select">
```
Comment (Sonic12040) [inline index.html]: Why is this wrapping div needed?

**Comment by Sonic12040** [comment]
I think a little more time should be put into this mobile layout. Most people visit these types of pages on their mobile devices. Perhaps visit your site on your mobile device, and see what is easy for you. What proves difficult? How do other sites handle mobile interfaces? What do you like about the interface of your favorite mobile applications?

**Review by Sonic12040 — CHANGES_REQUESTED** [review]

**Comment by fab514** [comment]
Colored css body and added the activity image.

**Comment by Sonic12040** [comment]
I would try something like:
```html

            <label name="label">
                Participants
                <br>
                <input class="input" type="number" name="participants" value="1" min="1" max="100">
            </label>
            <label name="label">
                    Activity Type
                <select name="type" class="select">
                    <option value="">Any</option>
                    <option value="cooking">Cooking</option>
                    <option value="education">Education</option>
                    <option value="social">Social</option>
                    <option value="recreational">Recreational</option>
                    <option value="busywork">Busywork</option>
                    <option value="charity">Charity</option>
                    <option value="relaxation">Relaxation</option>
                    <option value="diy">DIY</option>
                    <option value="music">Music</option>
                </select></label>
            <div class="search">
                <button>Search</button>
            </div>
        ```
        And use CSS like display: block to bump the select to the next line from the label.

**Review by Sonic12040 — CHANGES_REQUESTED** [review]
Just the item I mentioned above. This is looking great!

**Review by Sonic12040 — APPROVED** [review]

**Pull request merged**